### PR TITLE
Internal credential cache error ok

### PR DESCRIPTION
**Pull request by dbouchillon**
Fixes ZEN-23238

We can pass over this error.  simply means that 2 or more kinits
are attempting to authenticate at the same time.

### CODE REVIEW

**Comment by dbouchillon** [comment]
bump

File: txwinrm/util.py [hunk 1]
```diff
@@ -45,15 +45,18 @@
 _CONTENT_TYPE = {'Content-Type': ['application/soap+xml;charset=UTF-8']}
 _MAX_KERBEROS_RETRIES = 3
 _MARKER = object()
-_ENCRYPTED_CONTENT_TYPE={"Content-Type" : ["multipart/encrypted;protocol=\"application/HTTP-Kerberos-session-encrypted\";boundary=\"Encrypted Boundary\""]}
-_BODY="""--Encrypted Boundary
+_ENCRYPTED_CONTENT_TYPE = {"Content-Type": ["multipart/encrypted;protocol=\"application/HTTP-Kerberos-session-encrypted\";boundary=\"Encrypted Boundary\""]}
```
Comment (itshane) [inline txwinrm/util.py]: Can you split this string?

Comment (dbouchillon) [inline txwinrm/util.py]: done

File: txwinrm/util.py [hunk 2]
```diff
@@ -45,7 +45,9 @@
 _CONTENT_TYPE = {'Content-Type': ['application/soap+xml;charset=UTF-8']}
 _MAX_KERBEROS_RETRIES = 3
 _MARKER = object()
-_ENCRYPTED_CONTENT_TYPE = {"Content-Type": ["multipart/encrypted;protocol=\"application/HTTP-Kerberos-session-encrypted\";boundary=\"Encrypted Boundary\""]}
+_ENCRYPTED_CONTENT_TYPE = {"Content-Type": ["multipart/encrypted;protocol=\""
```
Comment (itshane) [inline txwinrm/util.py]: What do you think about this form:

**Pull request merged**